src/contracts/fund/participation/transactions/executeRequest.ts

```diff
@@ -1,46 +1,38 @@
-import { transactionFactory } from '~/utils/solidity';
-import { ensure } from '~/utils/guards';
-import { getRequest } from '../calls/getRequest';
-import { createQuantity, greaterThan } from '@melonproject/token-math/quantity';
-import { Contracts } from '~/Contracts';
-import { getToken } from '~/contracts/dependencies/token';
-import { getHub, getSettings, ensureIsNotShutDown } from '../../hub';
+import { executeRequestFor } from './executeRequestFor';
+import { getGlobalEnvironment } from '~/utils/environment';
+import { Address } from '@melonproject/token-math/address';
 
-const guard = async (params, contractAddress, environment) => {
```

Comment:
why are we removing the guards here?

Reply:
@schmidsi

Reply:
This is now just a wrapper function around `executeRequestFor`, which does the guards check then. I ran into problems with AMGU forwarding, I have the following hypothesis:

Given the two functions:
```solidity
function executeRequestFor(address requestOwner) public payable amguPayable {  /* ... do it */ }
function executeRequest() public payable amguPayable {
    executeRequestFor(msg.sender);
}
```

Where executeRequest() just calls executeRequestFor(), the amgu calculation is made twice and that leads to errors. So the hypothesis is:

** An amguPayable function cannot call another amguPayable function **

Actually, I think we can remove `executeRequest` completely from the contracts. --> Contracts should have as little code as possible. Every line of code enhances the possibility for bugs.

Reply:
ok @travs we should think of removing executeRequest as Simon said !

Reply:
@schmidsi yeah `executeRequest` was just for convenience but it is indeed a bit needless. I will remove it. AmguPayable was intentionally left off that function before this PR